test/langtools/tools/javac/annotations/typeAnnotations/failures/CantAnnotatePackages.out

```diff
@@ -1,5 +1,5 @@
-CantAnnotatePackages.java:14:13: compiler.err.cant.type.annotate.scoping.1: @TA
-CantAnnotatePackages.java:19:18: compiler.err.cant.type.annotate.scoping.1: @TA
-CantAnnotatePackages.java:20:19: compiler.err.cant.type.annotate.scoping.1: @TA
-CantAnnotatePackages.java:21:24: compiler.err.cant.type.annotate.scoping.1: @TA
+CantAnnotatePackages.java:14:18: compiler.err.type.annotation.inadmissible: (compiler.misc.type.annotation.1: @TA), java.lang, @TA java.lang.Object
```

Comment:
In this output, isn't `java.lang` the `1: symbol` and `@TA java.lang.Object` the `2: type` in the error message?
Won't the `to annotate a qualified type, write {1}.{2}` then output `java.lang.@TA java.lang.Object`, which isn't what we would want.
Am I parsing this incorrectly?

Reply:
I think you're parsing the arguments correctly, but when the arguments are formatted for the output the type is printed using its simple name, so we end up with:

```
test/langtools/tools/javac/annotations/typeAnnotations/failures/CantAnnotatePackages.java:14: error: type annotations are not admissible at this location: @TA
    @TA java.lang.Object of1;
                 ^
  (to annotate a qualified type, write java.lang.@TA Object)
  ```